src/app/components/App.tsx

```diff
@@ -292,6 +296,18 @@ const App = () => (
                 </SingleColumnLayout>
               )}
             />
+            {/* <Route
```

Comment:
You should remove the commented out route above then

Reply:
Or actually put it above the dashboard route to order and group similar routes together

Reply:
Updated